app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -21,13 +21,21 @@
 
 @import helper._
 @import uk.gov.hmrc.govukfrontend.views.html.components._
-@import uk.gov.nationalarchives.omega.editorial.models.{CorporateBody, EditSetRecord, User}
 @import uk.gov.hmrc.govukfrontend.views.viewmodels.fieldset.Legend
-@import uk.gov.nationalarchives.omega.editorial.models.{EditSetRecord, User, LegalStatus}
+@import uk.gov.nationalarchives.omega.editorial.models._
+@import uk.gov.nationalarchives.omega.editorial.forms.EditSetRecordFormValues
 @import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 
 @this(govukButton: GovukButton, govukInput: GovukInput, govukTextarea: GovukTextarea, govukFieldset: GovukFieldset, govukErrorSummary: GovukErrorSummary, govukSelect: GovukSelect)
-@(user: User, editSetName: String, title: String, legalStatusReferenceData: Seq[LegalStatus], corporateBodies: Seq[CorporateBody], editSetRecordForm: Form[EditSetRecord])(implicit messages: Messages, request: Request[AnyContent])
+@(
+  user: User,
+  editSetName: String,
+  title: String,
+  staticRecord: EditSetRecord,
```

Comment:
Why not just `record`; why `static`?

Reply:
I named this `staticRecord` to distinguish it from the form with "dynamic" values that change when this form is submitted, but I'm not sure about the wording, especially if other people find it unclear.

I'm happy to change it to be `record`, if you think that makes more sense?

Reply:
Up to you; but I'd suggest that we be consistent with `EditSetController`, either way.